hphp/runtime/vm/jit/vasm-gen-inl.h

```diff
@@ -75,13 +75,13 @@ namespace detail {
 
 template<class GenFunc>
 TCA vwrap_impl(CodeBlock& main, CodeBlock& cold, DataBlock& data,
-               CGMeta* meta, GenFunc gen, CodeKind kind) {
+               CGMeta* meta, GenFunc gen, CodeKind kind, bool fast) {
   CGMeta dummy_meta;
 
   auto const start = main.frontier();
 
   { // Finish emitting in this scope so that we can assert about `dummy_meta'.
-    Vauto vauto { main, cold, data, meta ? *meta : dummy_meta, kind };
+    Vauto vauto { main, cold, data, meta ? *meta : dummy_meta, kind, fast };
```

Comment:
This doesn't need to be an argument to the `Vauto` constructor.  Just add

```cpp
vauto.unit().fast = true;
```

after you construct it.